syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode{
    int32 errcode = 1;
    bytes ermsg = 2;
}

message GetFriendListRequest{
    uint32 userid = 1;
}

message GetFriendListResponse{
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendServiceRpc{
    rpc GetFriendList(GetFriendListRequest) returns(GetFriendListResponse);
}

